File: extractor-core/src/main/java/ru/mydesignstudio/database/metadata/extractor/parameters/MetadataSchema.java
package ru.mydesignstudio.database.metadata.extractor.parameters;

import com.google.common.collect.Lists;
import lombok.Data;

import java.util.List;

@Data
public class MetadataSchema {
  private String name;
  private List<MetadataTable> tables = Lists.newArrayList();
  private List<String> labels = Lists.newArrayList();
}

File: extractor-core/src/main/kotlin/ru/mydesignstudio/database/metadata/extractor/parameters/source/MetadataSchema.kt
package ru.mydesignstudio.database.metadata.extractor.parameters.source

/**
 * Describes some database to extract metadata from.
 */
class MetadataSchema(
        val name: String,
        val objects: List<MetadataObject>,
        val labels: MetadataSchemaLabels
) {

}